src/python/pants/option/global_options.py

```diff
@@ -401,13 +401,12 @@ def register_bootstrap_options(cls, register):
             help="Add these directories to PYTHONPATH to search for plugins.",
         )
         register(
-            "--spec-file",
+            "--spec-files",
```

Comment:
This is likely to break a lot of things. It's very commonly scripted to use `--spec-file`. I recommend adding a second flag `--spec-file` and deprecating it. Then, use `resolve_conflicting_options()`. See https://github.com/pantsbuild/pants/pull/10217/files#diff-043a29ac640559a4fdb3b888f5bc4dba for an example with `--changed-since`.